packages/esm-patient-common-lib/src/workspaces/workspaces.ts

```diff
@@ -103,40 +103,37 @@ function getTitleFromExtension(ext: ExtensionRegistration) {
   return ext.name;
 }
 
+/**
+ *
+ * @param name Name of the workspace
+ * @param ignoreChanges If set to true, the "unsaved changes" modal will never be shown, even if the `promptBeforeClosing` function returns true. The user will not be prompted before closing.
+ * @returns true if the workspace can be closed.
+ */
+export function getWhetherWorkspaceCanBeClosed(name: string, ignoreChanges: boolean = false) {
```

Comment:
Nit: I wish this was named differently. Don't have a better name just yet.

Reply:
Yeah good thought @denniskigen .

`shouldPromptBeforeClosingWorkspace`
`workspaceHasUnsavedChanges`

Also why is this exported? I wouldn't want this to be part of the API.